definitions/npm/react-router-dom_v4.x.x/flow_v0.38.x-v0.52.x/react-router-dom_v4.x.x.js

```diff
@@ -70,7 +70,7 @@ declare module 'react-router-dom' {
     goBack(): void,
     goForward(): void,
     canGo?: (n: number) => bool,
-    block(callback: (location: Location, action: HistoryAction) => boolean): void,
+    block(callback: (location: Location, action: HistoryAction) => boolean): ?string,
```

Comment:
Why it's different here than all the other fixes?

Reply:
The original file had this different structure and I wasn't sure if that was intentional (due to some Flow 0.52 < behavior) or not. So I decided to leave it as is.

Reply:
Good catch. Fix incoming.